charts/merlin/values.yaml

```diff
@@ -219,6 +219,8 @@ merlin:
       },
     ]
 
+    upiDocURL: "http://sample.doc.com"
```

Comment:
Can you make https://github.com/caraml-dev/universal-prediction-interface/blob/main/docs/api_markdown/caraml/upi/v1/index.md as default